code/nodes/optimizeModels/optimizeModels.q

```diff
@@ -1,21 +1,26 @@
-\d .automl
-
+// code/nodes/optimizeModels/optimizeModels.q - Optimize models node
+// Copyright (c) 2021 Kx Systems Inc
+//
 // Following the initial selection of the most promising model apply the user
-//   defined optimization grid/random/sobol if feasible.
-//   Ignore for keras/pytorch etc.
+// defined optimization grid/random/sobol if feasible.
+// Ignore for keras/pytorch etc.
+
+\d .automl
 
-/ @kind function
+// @kind function
 // @category node
-// @fileoverview Optimize models using hyperparmeter search procedures if 
+// @desc Optimize models using hyperparmeter search procedures if 
 //   appropriate, otherwise predict on test data
-// @param config {dict} Information related to the current run of AutoML
-// @param modelInfo {tab} Information about models applied to the data
+// @param config {dictionary} Information related to the current run of AutoML
+// @param modelInfo {table} Information about models applied to the data
 // @param bestModel {<} Fitted best model
-// @param modelName {sym} Name of best model
-// @param tts {dict} Feature and target data split into training/testing sets
-// @param orderFunc {func} Function used to order scores
-// @return {dict} Score, prediction and best model
-optimizeModels.node.function:{[config;modelInfo;bestModel;modelName;tts;orderFunc]
+// @param modelName {symbol} Name of best model
+// @param tts {dictionary} Feature and target data split into training/testing
+//   sets
+// @param orderFunc {fn} Function used to order scores
+// @return {dictionary} Score, prediction and best model
+optimizeModels.node.function:{[config;modelInfo;bestModel;modelName;tts;
```

Comment:
Can be single line if it'a a function input (even if it's over 80 chars)

Reply:
revert to long lines